src/main/scala/lc/core/config.scala

```diff
@@ -49,12 +50,7 @@ object Config {
   val allowedMedia: Set[String] = captchaConfig.flatMap(_.allowedMedia).toSet
   val allowedInputType: Set[String] = captchaConfig.flatMap(_.allowedInputType).toSet
 
-  private val random = new scala.util.Random(seed)
-
-  def getNextRandomInt(max: Int): Int =
-    random.synchronized {
-      random.nextInt(max)
-    }
+  HelperFunctions.seed = seed
```

Comment:
This won't update the seed of the instantiated generator. You need to do something like `HelperFunctions.random.setSeet(seed)`

Since `HelperFunctions.random` is private you will have to create a `synchronized setSeed()` method in `HelperFunctions` which does `random.setSeed()`